assets/js/googlesitekit/datastore/user/permissions.js

```diff
@@ -106,6 +107,35 @@ export const selectors = {
 		const { permissionError } = state;
 		return permissionError;
 	},
+
+	/**
+	 * Gets capabilities of the current user.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Object|undefined)} Capabilities object. Returns undefined if it is not loaded yet.
+	 */
+	getCapabilities( state ) {
+		const { capabilities } = state;
+		return capabilities;
+	},
+
+	/**
+	 * Checks if the current user has the specified capability or not.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} capability Capability name to check.
+	 * @return {(boolean|undefined)} TRUE if the current user has this capability, otherwise FALSE. If capabilities ain't loaded yet, returns undefined.
+	 */
+	hasCapability( state, capability ) {
+		const { capabilities } = state;
+		if ( capabilities ) {
+			return !! capabilities[ capability ];
+		}
```

Comment:
The docs say this returns `undefined` if `capabilities` aren't yet loaded, but there's no `return` statement after this `if` statement. We should explicitly return `undefined` rather than relying on the implicit `undefined` result of a function with no return value to improve readability.

Reply:
Updated.